code/modules/mob/living/resize.dm

```diff
@@ -4,6 +4,7 @@ var/const/RESIZE_GIGANTOR = 1.5
 var/const/RESIZE_BIG = 1.15
 var/const/RESIZE_NORMAL = 1
 var/const/RESIZE_SMALL = 0.85
+var/const/RESIZE_POCKET_PRINCE = 0.7
```

Comment:
```suggestion
var/const/RESIZE_POCKET_PRINCE = 0.75
```
?

Reply:
It make sense since the other sizes is increased/decreased by 0.15